hassio_aw2m/config.yaml

```
@@ -0,0 +1,33 @@
+name: AmbientWeather2MQTT
+version: "1.0.0"
+slug: hassio_aw2m
+description: This package listens for local data from Ambient Weather stations (such as the WS-2902C) and converts the incoming data to MQTT events.
+url: "https://github.com/neilenns/ambientweather2mqtt"
+startup: application
+boot: auto
+ports:
+  8132/tcp: 7000
+options:
+  MQTT_SERVER: ""
```

```suggestion
  MQTT_SERVER: "http://192.168.1.1:1883"
```

This will provide an example value in the UI to make it clear it's supposed to be an URL with a port number.